codegen/facelift/templates/IPCAdapter.template.cpp

```diff
@@ -80,6 +97,25 @@ struct {{interfaceName}}IPCAdapter::Impl {
 {{interfaceName}}IPCAdapter::{{interfaceName}}IPCAdapter(QObject* parent) :
     BaseType(facelift::InterfaceManager::instance(), parent)
 {
+#ifdef DBUS_IPC_ENABLED
```

Comment:
all that DBus specific code belongs to {{interfaceName}}IPCDBusAdapter

Reply:
I generally agree, but the next step is to remove the whole "local" specialization of the IPC adapter/proxies.
Since QtDBus has a built-in mechanism to treat the local async messages differently (similar to what "local" does)
so I have tried to treat as homogeneous as possible for the next patch.

Reply:
The "local" communication is needed on the platforms where QtDBus is not available.

Reply:
Then on platforms with QtDBus support there should be no manual "local" generation rather relying on QtDBus.
But that is subject of another discussion.